presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -328,6 +328,7 @@ public final class SystemSessionProperties
     // TODO: Native execution related session properties that are temporarily put here. They will be relocated in the future.
     public static final String NATIVE_AGGREGATION_SPILL_ALL = "native_aggregation_spill_all";
     private static final String NATIVE_EXECUTION_ENABLED = "native_execution_enabled";
+    private static final String NATIVE_SINGLE_WORKER_EXECUTION = "native_single_worker_execution";
```

Comment:
I tried singleNodeExecutionEnabled, but then realized it would cause confusion with [forceSingleNode](https://github.com/prestodb/presto/blob/1327a3e6c2025659760b8f94142dec0edc663485/presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java#L257)

Node can either be worker or coordinator, but what we want is explicitly worker. So `singleWorkerExecutionEnabled` makes more sense